bch-devsuite

```diff
@@ -200,6 +200,17 @@ def handle_smartbch():
     elif config.network in (Network.TESTNET, Network.MAINNET):
         config.smartbch = SmartBCHConf(enabled=True)
 
+def handle_bch_rpc_conf():
+    host = inquirer.text(message="BCH RPC host", default="http://bch-node")
```

Comment:
Where is this node? `http://bch-node`
message should be `"BCH RPC host *WITH PROTOCOL*"`

Reply:
Why not default to local node?

Reply:
It's the local node, it's hostname inside docker-compose network